test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -1018,6 +1019,58 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Offlined CPU API", func() {
+		It("should set offline cpus after deploy PAO", func() {
+			// Check default profile online cpus
+			for _, node := range workerRTNodes {
+				onlineOutput, err := nodes.ExecCommandOnMachineConfigDaemon(&node, []string{"cat", "/sys/devices/system/cpu/online"})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(string(onlineOutput)).To(ContainSubstring("0-3"))
+			}
+			By("Backing up the profile")
+			initialProfile := profile.DeepCopy()
+
+			// Create new performance with offlined
+			reserved := performancev2.CPUSet("0")
+			isolated := performancev2.CPUSet("1")
+			offlined := performancev2.CPUSet("2-3")
+			profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Enable UserLevelNetworking and add Devices in Profile")
+			profile.Spec.CPU = &performancev2.CPU{
+				Reserved: &reserved,
+				Isolated: &isolated,
+				Offlined: &offlined,
+			}
+			By("Updating the performance profile")
+			profiles.UpdateWithRetry(profile)
+
+			By("Waiting for MCP being updated")
+			mcps.WaitForCondition("worker-cnf", mcv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
```

Comment:
Sometimes reaches tiemout. Second reboot takes more time than first one and go beyond 10 minutes of timeout. Maybe extend timeout of MCP check?

Reply:
@marioferh Can you move this test case to functests/2_performance_update/updating_profile.go.  We generally keep all the performance profile update tests in this file.

Reply:
@mrniranjan thanks, in fact in updating_profile.go there are more similar test